epix/clustering.py

```diff
@@ -249,7 +249,8 @@ def _cluster(x, y, z, ed, time, ci,
 classifier['parenttype'] = ['None', 'None', 'None', 'Kr83[9.405]','Kr83[41.557]', 'None', 'None']
 classifier['creaproc'] = ['None', 'None', 'None', 'None', 'None','None', 'None']
 classifier['edproc'] = ['ionIoni', 'hadElastic', 'None', 'None','None', 'None', 'None']
-classifier['A'] = [0, 0, 4, 0, 0, 0, 0]
+classifier['A'] = [np.iinfo(np.int16).max, np.iinfo(np.int16).max, 4, np.iinfo(np.int16).max,\
```

Comment:
Could you put a comment citing the issue. It is not really obvious why you put the np.iinfo.

Reply:
Hi Daniel, Thank you for your comments. 

I wanted to use `np.inf` first but got some problems since it is a float and not an integer. `np.iinfo(np.int16).max` looks a bit ugly but this should be the maximal value that can fit into a int16.